app/build.gradle.kts

```diff
@@ -1,15 +1,30 @@
 import com.github.benmanes.gradle.versions.updates.DependencyUpdatesTask
 import com.github.benmanes.gradle.versions.updates.gradle.GradleReleaseChannel
+import io.gitlab.arturbosch.detekt.Detekt
 
 plugins {
     id("com.android.application")
     kotlin("android")
     kotlin("kapt")
     id("kotlin-parcelize")
+    id("io.gitlab.arturbosch.detekt") version Dependencies.Versions.detekt
     id("de.mannodermaus.android-junit5")
     id("com.github.ben-manes.versions") version Dependencies.Versions.dependencyUpdates
 }
 
+detekt {
+    buildUponDefaultConfig = true
```

Comment:
Adding `ignoreFailures = true` will make sure Detekt reports all issues - handy for when we add linting to CI.

Reply:
Theoretically, I want to have all failure fixed. I'll start working on it once ExoPlayer is merged.

Reply:
Yes but we want all the issues reported to GitHub so therefor it should be ignored.

Reply:
Ok :+1: